packages/cli/src/commands/setup/auth/providers/clerk.js

```diff
@@ -0,0 +1,47 @@
+// the lines that need to be added to App.{js,tsx}
+export const config = {
+  imports: [
+    `import { ClerkProvider, ClerkLoaded, useClerk } from '@clerk/clerk-react'`,
+  ],
+  init: `
+// You need to add two env variables: NEXT_PUBLIC_CLERK_FRONTEND_API
```

Comment:
Suggesting renaming the env vars, and removing the "NEXT_PUBLIC_" :)

Reply:
Ha of course! My bad - totally zombie'ing around like I'm still in Next.

Reply:
Also updated the docs PR to reflect this change.